From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Stultz <john.stultz@linaro.org>
Date: Fri, 3 Apr 2020 22:12:09 +0000
Subject: ANDROID: Switch CONFIG_QCOM_PDC CONFIG_QCOM_SCM and CONFIG_ARM_SMMU
 to be loaded as modules

To switch the QCOM_SCM driver to be loaded from a module,
we need to also switch QCOM_PDC and QCOM_ARM_SMMU to also
load from a module.

Bug: 153049053
Signed-off-by: John Stultz <john.stultz@linaro.org>
Change-Id: I0509628322b20dea5f7e16e973d93d369c0c2bff
[joneslee: Removed defconfig elements]
[joneslee: Removed init/Kconfig.gki elements]
Signed-off-by: Lee Jones <lee.jones@linaro.org>
---
 arch/arm64/configs/db845c_gki.fragment | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/configs/db845c_gki.fragment b/arch/arm64/configs/db845c_gki.fragment
--- a/arch/arm64/configs/db845c_gki.fragment
+++ b/arch/arm64/configs/db845c_gki.fragment
@@ -95,6 +95,9 @@ CONFIG_QCOM_GLINK_SSR=m
 CONFIG_RPMSG_QCOM_GLINK_RPM=m
 CONFIG_QCOM_SMD_RPM=m
 CONFIG_QCOM_RPMPD=m
+CONFIG_QCOM_PDC=m
+CONFIG_QCOM_SCM=m
+CONFIG_ARM_SMMU=m
 CONFIG_ARM_QCOM_CPUFREQ_HW=m
 # XXX Audio bits start here
 CONFIG_I2C_CHARDEV=m
